src/Native/pkg/dir.targets

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <File Include="@(NativeFile)" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <!-- Xplat builds strip symbols into a separate file in release mode. -->
+    <FindNeighboringSymbols Condition="'$(OSGroup)'!='Windows_NT' AND '$(ConfigurationGroup)'=='Release'">true</FindNeighboringSymbols>
+    <!-- Windows builds only emit pdbs in debug mode. -->
+    <FindNeighboringSymbols Condition="'$(OSGroup)'=='Windows_NT' AND '$(ConfigurationGroup)'=='Debug'">true</FindNeighboringSymbols>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(FindNeighboringSymbols)'=='true'">
+    <!-- On Windows, trim ".dll" before adding ".pdb". On xplat, the symbol extension is simply appended. -->
+    <NativeFileWithoutExtension Include="@(NativeFile)" Condition="'$(OSGroup)'!='Windows_NT'" />
+    <NativeFileWithoutExtension Include="@(NativeFile -> '%(RootDir)%(Directory)%(Filename)')" Condition="'$(OSGroup)'=='Windows_NT'" />
+
+    <File Include="@(NativeFileWithoutExtension -> '%(Identity)$(SymbolFileExtension)')">
+      <IsSymbolFile>true</IsSymbolFile>
+    </File>
+
+    <!-- Include placeholder pdb on non-Windows platforms to ensure package is treated like a symbol package. -->
+    <File Include="$(MSBuildThisFileDirectory)\_.pdb" Condition="'$(PackageTargetRuntime)'!='' AND '$(OSGroup)'!='Windows_NT'">
```

Comment:
Have we talked with MyGet/NuGet to understand if this sort of thing is reasonable to be doing?  I worry a little bit it's going to cause problems for downstream consumers.